### DISCUSSION TITLE
Messengers voting

### DISCUSSION THREAD

**Discussion by DiTsi** [eslupmi/impulse#192] [Polls]
Select preferred messenger you want to see in IMPulse

**Comment by levich** [comment]
Vk teams

**Comment by logicall** [comment]
express
https://express.ms

**Comment by DiTsi** [comment]
https://pachca.com/

**Comment by rich7e** [comment]
https://getcompass.ru

**Comment by nixargh** [comment]


**Reply by DiTsi** [reply]
Please add your like on comment above. pachca.com already in comments
